packages/dma-contracts/tasks/common/verification-utils.ts

```diff
@@ -100,3 +115,90 @@ export class ActionsDatabase {
     })
   }
 }
+
+export function validateActionHashes(
+  operationHashes: string[],
```

Comment:
`operationActionsHashes` ? I'm suggesting as I'm not familiar with that part of the codebase so it wasnt all obvious at first glance

Reply:
👍